utils/src/main/scala/za/co/absa/enceladus/utils/broadcast/LocalMappingTable.scala

```diff
@@ -66,24 +66,35 @@ object LocalMappingTable {
     val valueTypes = targetAttributes.flatMap(targetAttribute => {
       SchemaUtils.getFieldType(targetAttribute, mappingTableDf.schema)
     })
-    val structFields: Seq[StructField] = outputColumns.keys.toSeq.zip(valueTypes)
+
+    val structFields: Seq[StructField] = outputColumns.keys
+      .map(SchemaUtils.stripPathFromFieldName).toSeq
+      .zip(valueTypes)
       .map { case (name: String, fieldType: DataType) => StructField(name, fieldType) }
     val rowSchema = StructType(structFields)
 
     val targetColumns: Seq[Column] = targetAttributes.map(targetAttribute => col(targetAttribute))
-
-    val mappingColumns = targetColumns ++ keyFields.map(c => col(c))
+    val keyColumns = keyFields.map(c => col(c))
```

Comment:
At the risk of being seen as nitpicky 🤓. Future reader will wonder why `targetColumns` needs type declaration while `keyColumns` not. I believe it's good to keep consistency, at least in so adjacent code. (one or another, don't care which 😉 )